src/components/Button/Button.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { StyledButton, StyledButtonLink } from "./Button.styles";
+import BUTTON_VARIANTS from "./constants";
+
+const Button = ({ variant, children, ...rest }) => {
+  const { href, target } = rest;
+
+  if (href) {
+    const rel = (target === "_blank" && "noopener") || undefined;
```

Comment:
helper? I see the same logic in a few places.

Reply:
I'll make a utils dir